Automattic/wp-calypso
# [nudge-a-palooza] Upsell/design review wordads

This is last step before launching nudge-a-palooza test - our upsell pages were redesigned during a review process and we need to include these changes. this PR makes it happen.

Related post: p9jf6J-M4-p2

Test plan:

1. Go to `/feature/ads` and confirm it looks and scales like this:
* Desktop: https://user-images.githubusercontent.com/205419/43253751-f10f65ae-90c5-11e8-9c33-8aa6a1c09275.png
* 1000px: https://user-images.githubusercontent.com/205419/43253745-f066f55e-90c5-11e8-8ea0-4fbe0a2cd19f.png
* 650px: https://user-images.githubusercontent.com/205419/43253753-f1471aa8-90c5-11e8-997f-6e6bfbafd656.png

1. Go to `/feature/plugins` and confirm it looks and scales like this:
* Desktop: https://user-images.githubusercontent.com/205419/43253747-f09e8190-90c5-11e8-8d7a-7412b12f2113.png
* 1000px: https://user-images.githubusercontent.com/205419/43253746-f083553c-90c5-11e8-8ad8-0254854fea9c.png
* 650px: https://user-images.githubusercontent.com/205419/43253744-f04bbdca-90c5-11e8-870e-737b80c08c9a.png

1. Go to `/feature/store` and confirm it looks and scales like this:
* Desktop: https://user-images.githubusercontent.com/205419/43253946-805d6116-90c6-11e8-8224-cb07e310e0d0.png
* 1000px: https://user-images.githubusercontent.com/205419/43253752-f12b93be-90c5-11e8-8eb8-115b9dd0f907.png
* 650px: https://user-images.githubusercontent.com/205419/43253741-f00f4db8-90c5-11e8-809c-40b2d14abe69.png

1. Go to `/feature/themes` and confirm it looks and scales like this:
* Desktop: https://user-images.githubusercontent.com/205419/43253749-f0d5fc9c-90c5-11e8-902a-4fed1dc7785b.png
* 1000px: https://user-images.githubusercontent.com/205419/43253748-f0bb131e-90c5-11e8-944b-dfebc449f400.png
* 650px: https://user-images.githubusercontent.com/205419/43253750-f0f2c994-90c5-11e8-94a0-51159f3f51ca.png
